adapters/sovrn/sovrn.go

```diff
@@ -75,71 +100,54 @@ func addHeaderIfNonEmpty(headers http.Header, headerName string, headerValue str
-	if response.StatusCode == http.StatusNoContent {
+func (s *SovrnAdapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.RequestData, responseData *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if responseData.StatusCode == http.StatusNoContent {
 		return nil, nil
 	}
 
-	if response.StatusCode == http.StatusBadRequest {
+	if responseData.StatusCode == http.StatusBadRequest {
 		return nil, []error{&errortypes.BadInput{
-			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", response.StatusCode),
+			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", responseData.StatusCode),
 		}}
 	}
 
-	if response.StatusCode != http.StatusOK {
+	if responseData.StatusCode != http.StatusOK {
 		return nil, []error{&errortypes.BadServerResponse{
-			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", response.StatusCode),
+			Message: fmt.Sprintf("Unexpected status code: %d. Run with request.debug = 1 for more info", responseData.StatusCode),
 		}}
 	}
 
-	var bidResp openrtb2.BidResponse
-	if err := json.Unmarshal(response.Body, &bidResp); err != nil {
+	var bidResponse openrtb2.BidResponse
+	if err := json.Unmarshal(responseData.Body, &bidResponse); err != nil {
 		return nil, []error{&errortypes.BadServerResponse{
 			Message: err.Error(),
 		}}
 	}
 
-	bidResponse := adapters.NewBidderResponseWithBidsCapacity(5)
+	response := adapters.NewBidderResponseWithBidsCapacity(5)
 
-	for _, sb := range bidResp.SeatBid {
+	for _, sb := range bidResponse.SeatBid {
 		for i := 0; i < len(sb.Bid); i++ {
 			bid := sb.Bid[i]
 			adm, err := url.QueryUnescape(bid.AdM)
 			if err == nil {
 				bid.AdM = adm
-				bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{
-					Bid:     &bid,
-					BidType: openrtb_ext.BidTypeBanner,
-				})
-			}
-		}
-	}
 
-	return bidResponse, nil
-}
+				bidType := openrtb_ext.BidTypeBanner
 
-func preprocess(imp *openrtb2.Imp) (string, error) {
-	var bidderExt adapters.ExtImpBidder
-	if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
-		return "", &errortypes.BadInput{
-			Message: err.Error(),
-		}
-	}
+				if request.Imp[0].Video != nil {
+					bidType = openrtb_ext.BidTypeVideo
+				}
 
-	var sovrnExt openrtb_ext.ExtImpSovrn
-	if err := json.Unmarshal(bidderExt.Bidder, &sovrnExt); err != nil {
-		return "", &errortypes.BadInput{
-			Message: err.Error(),
+				response.Bids = append(response.Bids, &adapters.TypedBid{
+					Bid:     &bid,
+					BidType: bidType,
+				})
+			}
 		}
 	}
 
-	imp.TagID = getTagid(sovrnExt)
-
-	if imp.BidFloor == 0 && sovrnExt.BidFloor > 0 {
-		imp.BidFloor = sovrnExt.BidFloor
-	}
-
-	return imp.TagID, nil
+	return response, nil
 }
 
 func getTagid(sovrnExt openrtb_ext.ExtImpSovrn) string {
```

Comment:
I'm curious what is the purpose of this function? I know this is not in scope of this PR, but do you think you need it?

Reply:
I have inherited this code from someone who is no longer at Sovrn. But it appears that our original work with Prebid Server used a spelling on our tagid parameter that we quickly decided was wrong. 😄 

So the old version is deprecated, but not removed yet. Not sure how we will go forward with the actual removal, but we will figure that out.